### PR TITLE
Fix: check if value is not empty

### PR DESCRIPTION
**Pull request by hregis**
@eldy

### CODE REVIEW

File: htdocs/core/class/commonobject.class.php [hunk 1]
```diff
@@ -4635,7 +4635,7 @@ function showOptionals($extrafields, $mode='view', $params=null, $keyprefix='')
 					case "edit":
 						// GETPOST("options_" . $key) can be 'abc' or array(0=>'abc')
 						$getposttemp = GETPOST('options_'.$key, 'none');				// GETPOST can get value from GET, POST or setup of default values.
-						if (isset($getposttemp)) {
+						if (isset($getposttemp) && $getposttemp != '') {
```
Comment (eldy) [inline htdocs/core/class/commonobject.class.php]: What if getposttemp is an array ?
Also which kind of bug does this fix ?

Comment (hregis) [reply htdocs/core/class/commonobject.class.php]: @eldy 
add an extrafields in thirdparty by example, edit a thirdparty and add value in field, valid... the value is visible in view mode,  but edit again this thirparty, the value is hidden in edit mode

**Comment by hregis** [comment]
@eldy 
another fix version

**Comment by eldy** [comment]
OK.
Can you confirm you can still "clear" a value in edit mode ?

**Comment by hregis** [comment]
@eldy it's not "showOptionals" fonction that save values

**Pull request closed without merging**